src/ansys/fluent/core/fluent_connection.py

```diff
@@ -59,6 +62,68 @@ def run(self) -> None:
             cb()
 
 
+class AppendToFile:
```

Comment:
Just one comment I think: we could put all the transcript-related code in a new transcript.py.

Reply:
Done, Thank you.